sklearn/linear_model/passive_aggressive.py

```diff
@@ -85,6 +85,7 @@ def __init__(self, C=1.0, fit_intercept=True,
                                    n_jobs=n_jobs)
         self.C = C
         self.loss = loss
+        self.alpha = 1.0
```

Comment:
I just committed the change to revert the passing of the alpha parameter. The passive_aggressive.py is now unchanged.